src/Main.purs

```diff
@@ -29,6 +31,9 @@ intSentinel = IRTypeable.intSentinel
 renderers :: Array Doc.Renderer
 renderers = Renderers.all
 
+rendererForLanguage :: String -> Maybe Doc.Renderer
+rendererForLanguage = Renderers.rendererForLanguage
```

Comment:
Ah, I see -- yes, you no longer need to do the re-export from Main trick, as you can require any of our modules from TS.

Reply:
That means we can also remove `Main.renderers` in favor of `Renderers.all`?

Do I just have to name the definition file `Renderers.d.ts` and it'll work?

Reply:
Yes, I think that’s exactly how it works.